infra/README.md

```diff
@@ -0,0 +1,68 @@
+# Installation
+
+The environment can provisioned using the Bicep templates inclulded with the project and executed in a number of ways including:
+
+      1. Azure CLI
+      2. GitHub Actions
+      3. Azure DevOps
+      4. Azure Portal
+
+We will provide examples of the first two methods in this document. Manual provisioning using the Azure CLI or continuous delivery via GitHub Actions.
+
+<br/>
+
+## Deploying the Bicep templates using the Azure CLI
+
+#### :wrench: CREATING THE RESOURCE GROUP ####
+
+Before executing any of the following Bicep templates, you first need to login to your Azure account using the Azure CLI `az` command line tool:
+
```

Comment:
In the spirit of making things easy/inclusive for new to azure folk, should we add a paragraph about bicep.

Also explicitly add the following?
```
cd infra
```